docs/trees/guide.md

```diff
@@ -256,14 +256,6 @@ println(
 )
 ```
 
-> It's important to keep in mind that quasiquotes expand at compile-time into
```

Comment:
this sentence needs to be modified rather than removed. the original source is only preserved if there are no interpolated values, otherwise it will be reconstructed from the generated tree.

Reply:
Thanks, I have updated the message, and moved it after explaining interpolations. How about this?
https://github.com/scalameta/scalameta/pull/3572/commits/207ba9c19ea0c323df33aacd9e56217be1ca841e